packages/mds-geography-service/@types/index.ts

```diff
@@ -54,8 +54,8 @@ export type GetPublishedGeographiesOptions = GetGeographiesOptions &
     publishedAfter: Timestamp
   }>
 
-export type GeographyWithMetadataDomainModel<M extends {} = {}> = GeographyDomainModel &
-  Partial<Pick<GeographyMetadataDomainModel<M>, 'geography_metadata'>>
+export type GeographyWithMetadataDomainModel<M extends Record<string, any> = Record<string, any>> =
```

Comment:
I don't love this type, would be open to something which makes things a bit more optional.

Reply:
What's the difference between Record<string, any> and {}?

Reply:
`{}` is a literal empty object, with no properties at all. Record<string, any> is a fair bit more lenient